Libplanet/Net/Swarm.cs

```diff
@@ -1090,9 +1093,17 @@ await foreach (var pair in completedBlocks)
                             totalBlocksToDownload
                         );
 
-                        IAsyncEnumerable<Tuple<long, HashDigest<SHA256>>> blockHashes =
-                            GetBlockHashes(peer, locator, null, cancellationToken);
-                        await foreach (Tuple<long, HashDigest<SHA256>> pair in blockHashes)
+                        List<Tuple<long, HashDigest<SHA256>>> blockHashes =
+                            await GetBlockHashes(peer, locator, null, cancellationToken)
+                                .ToListAsync(cancellationToken);
```

Comment:
How about keeping `blockHashes` as `IAsyncEnumerator` and using `AnyAsync()` & `FirstAsync()` instead of materializing by `ToListAsync()`?

Reply:
Changed it as you said and amended the commit.